### PR TITLE
Remove log4j

### PR DESCRIPTION
**Pull request by frenchy64**
I think https://github.com/threatgrid/ctia/pull/1190 could have (intended to?) remove log4j after adding [log4j-over-slf4j](https://www.slf4j.org/legacy.html#log4j-over-slf4j).

<a name="qa">[§](#qa)</a> QA
============================

No QA is needed.

<a name="release-notes">[§](#release-notes)</a> Release Notes
=============================================================

```
intern: log4j 2 
```

<a name="squashed-commits">[§](#squashed-commits)</a> Squashed Commits
======================================================================

### CODE REVIEW

**Review by marioaquino — APPROVED** [review]

File: project.clj [hunk 1]
```diff
@@ -157,6 +159,8 @@
                  [com.fasterxml.jackson.dataformat/jackson-dataformat-cbor ~jackson-version] ;; bump cheshire, align with jackson-databind
                  [com.fasterxml.jackson.core/jackson-databind ~jackson-databind-version] ;; bump onyx-kafka and others
                  [zookeeper-clj "0.9.4"]
+                 [org.apache.logging.log4j/log4j-core ~log4j2-version] ;;log4j2
+                 [org.apache.logging.log4j/log4j-1.2-api ~log4j2-version] ;; log4j 1=>2 bridge
```
Comment (msprunck) [inline project.clj:163]: Are we still using log4j? I replaced log4j by logback 2 years ago with this [commit](https://github.com/threatgrid/ctia/commit/3b7d6c25ac408a7acd83f49e30127514f8705509). Even if log4j is used by a 3rd party library, it is redirected to slf4j with `org.slf4j/log4j-over-slf4j`. 

If this update is still needed could you make sure logs produced from Trapperkeeper and clojure.tools.logging are still there and handled by logback?

Comment (frenchy64) [reply project.clj:163]: > Are we still using log4j?

That's what I wanted to ask you about. I'd prefer to just remove it. I wasn't sure why log4j is still on the classpath, just an oversight? Apparently the way to use the redirection is to remove log4j from the classpath: https://www.slf4j.org/legacy.html#log4j-over-slf4j

Comment (frenchy64) [reply project.clj]: I changed this PR to remove log4j, for the above reason.

Comment (frenchy64) [reply project.clj]: I think [this PR](https://github.com/threatgrid/ctia/pull/1190) could have (intended to?) remove log4j after adding [log4j-over-slf4j](https://www.slf4j.org/legacy.html#log4j-over-slf4j).

**Review by msprunck — APPROVED** [review]
LGTM!

**Review by gbuisson — APPROVED** [review]

**Pull request merged**